content/applications/hr/employees/retention_report.rst

```diff
@@ -0,0 +1,102 @@
+
+Next, click the :guilabel:`Measures` :icon:`fa-caret-down` button in the upper-left corner,
+revealing a drop-down menu. Click :guilabel:`# New Employees`, :guilabel:`Annual Employee Budget`,
+:guilabel:`Fuel Card`, and :guilabel:`Wage` in the list, to deselect these metrics and hide them in
+the table. Then, click :guilabel:`Count` at the bottom of the list to enable that metric.
+
+Click away from the drop-down menu to close it. Now the report shows all the employees who left the
+company (:guilabel:`# Departure Employee`), as well as the total number of employees
+(:guilabel:`Count`), for the :guilabel:`Last 365 Days`.
+
+To compare the data for the current year with the previous year, click the :icon:`fa-caret-down`
+:guilabel:`(down arrow)` in the search bar, revealing multiple filter and grouping options. Click
+:guilabel:`Last 365 Days` in the :icon:`fa-filter` :guilabel:`Filters` column, to turn off that
+filter. Then click :guilabel:`Date`, and click the current year (in this example, :guilabel:`2024`)
+from the resulting drop-down menu.
+
+Once a selection is made beneath :guilabel:`Date` in the :icon:`fa-filter` :guilabel:`Filters`
+column, a :icon:`fa-adjust` :guilabel:`Comparison` column appears. Click :guilabel:`Date: Previous
+Year` in the new column, then click off of the drop-down menu to close it.
+
+.. note::
+   In Odoo, in order to access the :icon:`fa-adjust` :guilabel:`Comparison` column, a specific time
+   *other than* :guilabel:`Last 365 Days` *must* be selected. If not, the :icon:`fa-adjust`
+   :guilabel:`Comparison` column is **not** visible.
+
+Now the pivot table displays the total number of employees who left the company (:guilabel:`#
+Departure Employee`), as well as the total number of employees (:guilabel:`Count`) in the columns.
+These are further divided by the two different years, and also displays the :guilabel:`Variation`
+between the two.
+
+The rows display the departments, and lists each individual employee for each department, in the
+rows.
+
+For a more concise view of this report, click :icon:`fa-minus-square-o` :guilabel:`Total` at the top
+of the rows, to collapse the rows. Now the table presents the total number of employees who left the
+company for both years, compared to the total number of employees for both years, as well as the
+difference, in a percentage.
+
+.. example::
+   In this example, three employees out of 83 left in 2023, and eight employees out of 202 left in
+   2024. There was a :guilabel:`166.67%` increase in the employees who left in 2024 as compared to
+   2023. Additionally, there was a :guilabel:`143.37%` increase in the total number of employees in
+   2024. 2024 as compared to 2023.
+
+   .. image:: retention_report/comparison-years.png
+      :align: center
+      :alt: The report modified to show the difference between two years of employees who left.
+
+To view more detailed rates for each department, click :icon:`fa-plus-square` :guilabel:`Total`,
```

Comment:
At this point in the workflow, there are two "+ Total"s available, and when I click the one that is higher up and I follow the other instructions, what I see does not match the screenshot, but when I click the lower one on the left and follow the other instructions, what I see matches the screenshot. It might be worth revising this line to clarify which "+ Total" to click!